lite/examples/object_detection/android/app/src/androidTest/java/org/tensorflow/lite/examples/detection/DetectorTest.java

```diff
@@ -14,151 +14,150 @@
-    canvas.drawBitmap(loadImage("table.jpg"), frameToCropTransform, null);
-    final List<Recognition> results = detector.recognizeImage(croppedBitmap);
-    final List<Recognition> expected = loadRecognitions("table_results.txt");
-
-    for (Recognition target : expected) {
-      // Find a matching result in results
-      boolean matched = false;
-      for (Recognition item : results) {
-        RectF bbox = new RectF();
-        cropToFrameTransform.mapRect(bbox, item.getLocation());
-        if (item.getTitle().equals(target.getTitle())
-            && matchBoundingBoxes(bbox, target.getLocation())
-            && matchConfidence(item.getConfidence(), target.getConfidence())) {
-          matched = true;
-          break;
-        }
-      }
-      assertThat(matched).isTrue();
-    }
-  }
-
-  // Confidence tolerance: absolute 1%
-  private static boolean matchConfidence(float a, float b) {
-    return abs(a - b) < 0.01;
-  }
-
-  // Bounding Box tolerance: overlapped area > 90% of each one
-  private static boolean matchBoundingBoxes(RectF a, RectF b) {
-    float areaA = a.width() * a.height();
-    float areaB = b.width() * b.height();
-
-    RectF overlapped =
-        new RectF(
-            max(a.left, b.left), max(a.top, b.top), min(a.right, b.right), min(a.bottom, b.bottom));
-    float overlappedArea = overlapped.width() * overlapped.height();
-    return overlappedArea > 0.9 * areaA && overlappedArea > 0.9 * areaB;
-  }
-
-  private static Bitmap loadImage(String fileName) throws Exception {
-    AssetManager assetManager =
-        InstrumentationRegistry.getInstrumentation().getContext().getAssets();
-    InputStream inputStream = assetManager.open(fileName);
-    return BitmapFactory.decodeStream(inputStream);
-  }
-
-  // The format of result:
-  // category bbox.left bbox.top bbox.right bbox.bottom confidence
-  // ...
-  // Example:
-  // Apple 99 25 30 75 80 0.99
-  // Banana 25 90 75 200 0.98
-  // ...
-  private static List<Recognition> loadRecognitions(String fileName) throws Exception {
-    AssetManager assetManager =
-        InstrumentationRegistry.getInstrumentation().getContext().getAssets();
-    InputStream inputStream = assetManager.open(fileName);
-    Scanner scanner = new Scanner(inputStream);
-    List<Recognition> result = new ArrayList<>();
-    while (scanner.hasNext()) {
-      String category = scanner.next();
-      category = category.replace('_', ' ');
-      if (!scanner.hasNextFloat()) {
-        break;
-      }
-      float left = scanner.nextFloat();
-      float top = scanner.nextFloat();
-      float right = scanner.nextFloat();
-      float bottom = scanner.nextFloat();
-      RectF boundingBox = new RectF(left, top, right, bottom);
-      float confidence = scanner.nextFloat();
-      Recognition recognition = new Recognition(null, category, confidence, boundingBox);
-      result.add(recognition);
-    }
-    return result;
-  }
-}
+//package org.tensorflow.lite.examples.detection;
```

Comment:
@sayannath Check this example if it can help you:
https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-arch-core-release/camera/camera2/src/androidTest/java/androidx/camera/camera2/UseCaseCombinationTest.java

Reply:
Noted!